CONTRIBUTING.md

```diff
@@ -0,0 +1,50 @@
+# Contributing to IBM i Pipeline Steps
+
+## Building
+### Prerequisites
+- Java 64 bits JDK 17 or 21 ([Adoptium's JDK](https://adoptium.net/fr/temurin/releases/?arch=x64&package=jdk&version=21) are a good option)
```

Comment:
You use the French URL. Maybe use https://adoptium.net/temurin/releases/?arch=x64&package=jdk&version=21

Reply:
Oh my, you're right 😅 That will teach me to copy and paste URLs late in the evening.